changelog.json

```
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.8.4-beta1": [
+      "[New] Squash commits in the history tab - #2507",
+      "[Fixed] Remove extra space in drag & drop tooltips - #12327",
+      "[Fixed] Remove taskbar progress indicator - #9489",
+      "[Improved] Add Links to Official Apple Silicon Installers - #12306. Thanks @say25!",
+      "[Improved] Upgrade Python dependency to version 3.x - #12203. Thanks @ferenczy!"
```

The last two don't impact users so can be removed, and the taskbar progress indicator could be under a `[Removed]` release note type, no?
```suggestion
      "[Fixed] Remove extra space in drag & drop tooltips - #12327",
      "[Removed] Remove taskbar progress indicator - #9489"
```

---

On the non impacting ones, I left them since they had "Thanks" on them. Do you think we should still not have them?

:P didn't remember there was a [Removed]. Thanks!